ext/wadl-doclet/src/main/java12/org/glassfish/jersey/wadl/doclet/ResourceDoclet.java

```diff
@@ -190,7 +190,6 @@ public boolean run(DocletEnvironment docEnv) {
                     }
                     result.getDocs().add(classDocType);
                     success = DocletUtils.createOutputFile(optionOutput.getValue(), docProcessor, result);
-                    
```

Comment:
why?

Reply:
You are right: this was not necessary. [Checkstyle rule](https://github.com/eclipse-ee4j/jersey/blob/master/etc/config/checkstyle-verify.xml#L132-L136) was incorrect understood by me. Solved here and in all similar occurencies